io/src/main/scala/sbt/io/RichURI.scala

```diff
@@ -50,7 +50,9 @@ class RichURI(uri: URI) {
     }
 }
 
-object RichURI {
+object RichURI extends URIExtra
+
+trait URIExtra {
```

Comment:
I'm being consistent with sbt's use of trait in other places like `ProjectExtra` - https://github.com/sbt/sbt/blob/0.13.8/main/src/main/scala/sbt/Project.scala